airflow-core/src/airflow/decorators/setup_teardown.py

```diff
@@ -30,6 +29,8 @@
 
 
 def setup_task(func: Callable) -> Callable:
```

Comment:
This should go in the Task SDK

Reply:
Yeah, will pick task sdk ones in a follow up.